LearnDjango/tests/testAssignment.py

```diff
@@ -0,0 +1,58 @@
+from django.test import TestCase
+from django.test import Client
+from TAInformation.models import Course, User
```

Comment:
Line 3 and 4 should be combined. also see other comment about taAssignment

Reply:
Ye, I can do that